manager/allocator/networkallocator/networkallocator.go

```diff
@@ -21,6 +22,13 @@ const (
 	// default if a network without any driver name specified is
 	// created.
 	DefaultDriver = "overlay"
+	// PredefinedNetworkLabel is the swarmkit label attached to a network object
+	// representing one of the predefined networks known to be statically
+	// created on the cluster nodes. These objects are populated in the store
+	// at cluster creation solely in order to support running services on the
+	// nodes' predefined networks. As a consequence, these networks need not
+	// to be exposed via public APIs.
+	PredefinedNetworkLabel = "com.docker.swarm.predefinedNetwork"
```

Comment:
In that case, there is no reason for this label to be defined in the allocator code, and for its name to be tied to network. I am keen to change it to
```
// Predefined is a label to mark a swarm resource as a predefined resource
Predefined = "com.docker.swarm.predefined"
```
and move the definition in api/naming pkg